src/utils/gas.ts

```diff
@@ -0,0 +1,176 @@
+import { Provider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+import { GnosisSafe } from '../../typechain'
+import { sameString } from './'
+import { EMPTY_DATA } from './constants'
+
+// Parses the result from the error message (GETH, OpenEthereum/Parity and Nethermind) and returns the data value
+const getDataFromNodeErrorMessage = (errorMessage: string): string | undefined => {
```

Comment:
Does this work with hardhat (or ganache)?

Reply:
No, it doesn't. That's the issue I'm having now